nextjs/next.config.js

```diff
@@ -24,11 +25,21 @@ const nextConfig = {
 
   // TODO: remove this code once Proxy is no longer needed
   images: { unoptimized: process.env.NODE_ENV === 'development' },
+
+  async rewrites() {
+    return [
+      {
+        source: `/api/:path*`,
+        destination: `https://web:3000/api/:path*`,
```

Comment:
I think we're going to want to make this domain dynamic, but can be in a followup PR